### ISSUE TITLE
Sync'd bookmark favicon is shown only after opening the page

### ISSUE THREAD

**Issue by srirambv** [brave/browser-ios#762]
Description:
Sync'd bookmark favicon is shown only after opening the page

**Did you search for similar issues before submitting this one?**
Yes

**Expected behavior:**
Should load the favicon of the site when the bookmark is sync'd

- Device (iPhone5, iPhone6s plus, iPad 3, ?):
iPhone 6 (iOS 10)

- Brave Version:
1.4.1

- Steps to reproduce:
    1. Enable sync on desktop and add few bookmarks 
    2. Sync iPhone with the laptop using the QRCode
    3. Bookmarks are synced without favicons. Favicons are loaded only when the bookmark is loaded

** Website problems only:
- did you check with Brave Shields down?
- did you check in Puffin/Dolphin/iCab/Maxthon (UIWebView-based browsers)?

- Screenshot if needed:
![image](https://cloud.githubusercontent.com/assets/17010094/24329691/d2ba96ea-1205-11e7-96a0-17e5132e26d3.png)

- Any related issues:

**Comment by jhreis** [comment]
We currently now have a default placeholder icon. Although not ideal, this will be v1 for sync, so pushing this.

**Comment by srirambv** [comment]
Closing in favour of https://github.com/brave/sync/issues/106
